plugin/pkg/admission/imagepolicy/gencerts.sh

```diff
@@ -85,18 +85,22 @@ limitations under the License.
 
 EOF
 
-echo "// This file was generated using openssl by the gencerts.sh script" >> $outfile
-echo "// and holds raw certificates for the imagepolicy webhook tests." >> $outfile
-echo "" >> $outfile
-echo "package imagepolicy" >> $outfile
+
+{
+	echo "// This file was generated using openssl by the gencerts.sh script"
+	"// and holds raw certificates for the imagepolicy webhook tests."
+	""
+	"package imagepolicy"
+} >> $outfile
```

Comment:
I am unfamiliar with this idiom, what shellcheck failure is this addressing?

Reply:
[SC2129](https://github.com/koalaman/shellcheck/wiki/SC2129) - _"Consider using { cmd1; cmd2; } >> file instead of individual redirects."_

It's self-admitted by the shellcheck folk to be stylistic. 

However, a change is still needed. Each line should retain the echo command.
```
{
	echo "// This file was generated using openssl by the gencerts.sh script"
	echo "// and holds raw certificates for the imagepolicy webhook tests."
	echo ""
	echo "package imagepolicy"
}  >> "$outfile"
```

Reply:
@mrbobbytables  thanks, I'll fix it now